tests/cpp/test_mma.cpp

```diff
@@ -388,8 +388,10 @@ TEST_P(HopperRS, SingleTile) {
   {
     auto s = mma_utils::MmaSwizzler::scheduleMmaOutputAllocation(
         tv2c->getLoopDomain());
-    tv2c->setLoopDomain(s.as<IterDomain*>());
```

Comment:
As we previously discussed in [Representing ldmatrix](https://docs.google.com/document/d/1C7FJW_ySGgBLCYuZR2xM67SQJGIxlHwACpnSweImX78/edit?usp=sharing), we need both the read domain and write domain for MmaOp. And the read and write domain will not match. So, there is no correct answer of what the loop domain of the mma output should be unless we implement that document.

Previously, we are scheduling the loop domain as the write domain, which allows us to have multiple tiles in register (but we are not using this capability because we always just have one tile in all unit tests). But because this PR starts to infer instead of hard-code the strides, we really need to have a correct read domain, so I decide to change the loop domain from write domain to read domain. With this change, we will be able to support multiple tiles in smem, but not in registers. But for now, we can always make sure there is only one tile in register by inlining most.

Reply:
Added